api/managementpb/json/managementpb.json

```diff
@@ -2528,6 +2528,12 @@
                       "type": "string",
                       "x-order": 0
                     },
+                    "collections_limit": {
+                      "type": "integer",
+                      "format": "int32",
+                      "title": "Collections limit. Only get Databases and collection stats if the total number of collections in the server\nis less than this value. 0: no limit",
```

Comment:
`description` instead of `title`, for sake of consistency?

Reply:
This is auto generated by swagger libs